test/test_alexa_integration_express.js

```diff
@@ -211,7 +208,7 @@ describe("Alexa", function() {
       it("requires a cert header", function() {
         return request(testServer)
           .post('/testApp')
-          .expect(401).then(function(res) {
+          .expect(400).then(function(res) {
```

Comment:
I know this is unrelated, but can we please track this one down and make sure that's not a bug?

Reply:
I checked the alexa-verifier-middleware and for this change [(38ec0c)](https://github.com/alexa-js/alexa-verifier-middleware/commit/38ec0c37b15c75ff93cbb650a27c4dda48b62457). There was no explanation for this change, but they also [fixed their tests](https://github.com/alexa-js/alexa-app/pull/310#pullrequestreview-92028980) and released v1.0.1 with this change.